lib/rhc/commands/scp.rb

```diff
@@ -32,6 +32,22 @@ def run(_, action, local_path, remote_path)
       raise RHC::ArgumentNotValid.new("'#{action}' is not a valid argument for this command.  Please use upload or download.") unless action == 'download' || action == 'upload'
       raise RHC::FileOrPathNotFound.new("Local file, file_path, or directory could not be found.") unless File.exist?(local_path)
 
+      if options.ssh
+        warn "User specified a ssh executable"
+        if windows?
+          warn("On Windows, file transfers to/from a gear can be completed with many different third-party tools such as FileZilla or WinSCP.\n" \
+            "Alternatively, do not specify a custom ssh executable to transfer files through the rhc tool's internal ssh implementation.")
```

Comment:
It's a minor thing, but "do not do X to do Y" is ambiguous and makes my brain hurt a tiny bit; something like, "Alternatively, omit the --ssh flag in order to transfer files using the rhc tool's internal SSH implementation" might read better.  I don't feel too strongly about this point though.